includes/definitions.inc.php

```diff
@@ -3364,6 +3379,11 @@
 $config['device_types'][$i]['type'] = 'appliance';
 $config['device_types'][$i]['icon'] = 'appliance.png';
 
+$i++;
+$config['device_types'][$i]['text'] = 'Proxy';
```

Comment:
I agree that we don't want the list to get too long.  Why would we use "Proxy" over "Network" here?